src/components/forms/TextArea.tsx

```diff
@@ -0,0 +1,126 @@
+import styled from '@emotion/styled';
+import { CSSProperties, TextareaHTMLAttributes } from 'react';
+
+import { FullSpinner } from '../index';
+
+import { useFieldsContext } from './context/FieldsContext';
+import {
+  RenderAddon,
+  RootInput,
+  GroupStyled,
+  LabelStyled,
+  LeadingAddonStyled,
+  LeadingInlineAddonStyled,
+  TrailingInlineAddonStyled,
+  TrailingAddonStyled,
+} from './styles';
+
+const TextAreaStyled = styled.textarea`
+  padding: 0;
+  flex: 1 1 0%;
+  border: none;
+  position: relative;
+  outline: none;
+`;
+
+export interface TextAreaProps
+  extends TextareaHTMLAttributes<HTMLTextAreaElement> {
+  variant?: 'default' | 'small';
+
+  leadingAddon?: RenderAddon;
+  trailingAddon?: RenderAddon;
```

Comment:
Those 2 props should not exist on the TextArea component. It doesn't really make sense to have them because the component expands vertically and it really doesn't look nice to have them / it wastes a lot of screen real estate.